components/layout/grid/src/gridItem/index.js

```diff
@@ -1,10 +1,30 @@
 import PropTypes from 'prop-types'
 import cx from 'classnames'
 
-import {BASE_CLASS, CELL_NUMBERS} from '../settings'
+import {BASE_CLASS, CELL_NUMBERS, BREAKPOINTS} from '../settings'
+
+const getColSpanClassNamesTransform = ({colSpan, ...otherProps}) => {
+  const className = []
+  if (CELL_NUMBERS.includes(colSpan) && otherProps.xxs === undefined) {
```

Comment:
what's the point of xxs being undefined? Maybe a descriptive fn or a comment would help to understand what's happening here